toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt

```diff
@@ -51,6 +62,16 @@ public val MapViewInteractionOptionDefaults: MapViewInteractionOptions = MapView
  * @param locationDisplay the [LocationDisplay] used by the composable [com.arcgismaps.toolkit.geocompose.MapView]
  * @param mapViewInteractionOptions the [MapViewInteractionOptions] used by this composable [com.arcgismaps.toolkit.geocompose.MapView]
  * @param onViewpointChanged lambda invoked when the viewpoint of the composable MapView has changed
+ * @param isInteracting lambda invoked when the user is currently interacting with the composable MapView
```

Comment:
```suggestion
 * @param isInteracting lambda invoked when the user is currently interacting with the composable [MapView]
```

At first, I was considering to suggest using `[com.arcgismaps.toolkit.geocompose.MapView]` for the doc in this PR but I feel like it would be somewhat overwhelming especially when the doc is previewed in Android Studio: 

![image](https://github.com/Esri/arcgis-maps-sdk-kotlin-toolkit/assets/14892474/84548981-c783-4431-8b9e-3b39a0d14702)

Maybe (in this / a future issue), entirely remove the import `import com.arcgismaps.mapping.view.MapView` and instead create the MapView using:
```kotlin
val mapView = remember { MapView(context) } // Before
val mapView = remember { com.arcgismaps.mapping.view.MapView(context) } // After
```

This way we can use `[MapView]` in the docs to freely reference the composable MapView rather than the SDK MapView. Thoughts? @puneet-pdx @gunt0001